es/2/interfaces.md

```diff
@@ -1,6 +1,77 @@
 <!-- Add translation for the following page: https://vyper.fun/#/2/interfaces
 Do NOT change the code below. The below code runs the code editor -->
 
+# Capítulo 8: Interfaces
+
+En el capítulo anterior, completamos el contrato para la batalla pokemon. Ahora, para luchar con los pokemones, el contrato de entrenador (¿recuerdas el primer contrato en el que cada entrenador poseía un pokemon?) debe interactuar con el contrato de la batalla de pokemones.
+
+Para ello utilizaremos las Interfaces.
+
+## Interfaces
+
+Una interfaz es un conjunto de definiciones de funciones utilizadas para permitir la comunicación entre contratos inteligentes. Una contrato de interfaz define todas las funciones disponibles externamente de ese contrato (como no podemos acceder a las funciones privadas fuera del contrato, no tiene sentido añadirlas a la interfaz). Al importar una interfaz, tu contrato sabe cómo llamar a esas funciones en otros contratos.
+
+Así que, para nuestro caso de uso, necesitamos crear una interfaz para el contrato de batalla pokemon en el contrato de entrenador. Esto permitirá que el contrato de entrenador se comunique con el contrato de batalla pokemon.
+
+## Usando Interfaces
+
+Las Interfaces se pueden añadir a los contratos, ya sea a través de definición en línea o importándolos de un archivo separado.
+
+La palabra `interface` se utiliza para definir una interfaz externa en línea:
+
+    interface Car:
+        def start() -> bool: nonpayable
+        def getSpeed() -> uint256: view
+
+La interfaz definida `Car` puede ser utilizada para hacer llamadas externas, dada una dirección de contrato:
+
+    @external
+    def Driver(some_address: address):
+        Car(some_address).start()
+
+El nombre de la interfaz (`Car`) también puede utilizarse como un tipo para las variables de almacenamiento. A continuación, se asigna un valor de dirección a la variable para acceder a esa interfaz:
+
+    car: Car
+
+    @external
+    def __init__(car_address: address):
+        self.car = Car(car_address)
+
+    @external
+    def getCarSpeed():
+        self.car.getSpeed()
+
+La especificación de la anotación `payable` o `nonpayable` indica que la llamada realizada al contrato externo podrá alterar el contrato de almacenamiento/datos, mientras que la `view`, la llamada `pure` NO podrá ser alterar el contrato de almacenamiento/datos. Además, `payable` permite enviar un valor distinto de cero junto con la llamada (lo que significa que se puede enviar ETH cuando se llama a una función payable).
```

Comment:
Not sure if this the problem with the google translate, but the English translation is a bit confusing for `view`, `pure` sentence.

**Spanish**
```
La especificación de la anotación `payable` o `nonpayable` indica que la llamada realizada al contrato externo podrá alterar el contrato de almacenamiento/datos, mientras que la `view`, la llamada `pure` NO podrá ser alterar el contrato de almacenamiento/datos.
```

**English (Generated)**
```
Specifying annotation `payable` or ` nonpayable` indicates that the call made to the external contract may alter the contract storage / data, while `view` , called ` pure` NO may be altering the storage contract / data.
```

**English (Expected)**
```
Specifying `payable` or `nonpayable` annotation indicates that the call made to the external contract will be able to alter contract storage/data, whereas the view, pure call will NOT be able to alter contract storage/data.
```

The part `while `view` , called ` pure` NO may be altering the storage contract / data` in **English (Generated)** is a bit confusing/not clear. (This maybe the issue with my translator)

Please change the **Spanish** translation so that it clearly conveys `whereas the view, pure call will NOT be able to alter contract storage/data.`  :smile:

Reply:
What do you think about this?

> La especificación de la anotación `payable` o `nonpayable` que una llamada realizada desde un contrato externo podrá alterar el almacenamiento/datos del contrato, mientras que las llamadas anotadas como `view` o `pure` no serán capaces de alterar el almacenamiento/datos del contrato.

Reply:
LGTM